server/routes/ticket.js

```diff
@@ -1,29 +1,20 @@
 import express from 'express';
 import validate from 'express-validation';
 import paramValidation from './validation/ticket';
-
 import ticketController from '../controllers/ticket';
 
 const router = express.Router();  // eslint-disable-line new-cap
 
 router.route('/all')
-
   .get(validate(paramValidation.getTicketsByDate), ticketController.getTicketsByDate);
 
 router.route('/:id')
-
-  /** GET /api/v1/ticket/:id - Get a ticket by Id */
   .get(validate(paramValidation.byId), ticketController.getById)
-
-  /** PUT /api/v1/ticket/:id - Update a ticket */
+  .post(validate(paramValidation.byId), ticketController.close)
```

Comment:
I went with the logic that I wasn't going to be deleting the ticket, so therefor it wasn't a DELETE. Why would you want the DELETE over POST?